en/docs/reference/product-apis/devportal-apis/devportal-v3/devportal-v3.yaml

```diff
@@ -5965,6 +5965,8 @@ components:
       in: query
       description: |
         Application Group Id
+
```

Comment:
Fix the same here as well: https://github.com/wso2/docs-apim/pull/9410#discussion_r2188894005